Insights/InputEntity_Account/WindowsSigninActivity.yaml

```diff
@@ -210,5 +210,5 @@ Insights:
   Type: LineChart
 
  AdditionalQuery:
-  Text: "See All Windows Logons"
+  Text: "See all windows sign-ins"
```

Comment:
Windows should be capitalized as it is a proper noun

Reply:
Done